cmd/thv-operator/api/v1alpha1/mcpgroup_types.go

```diff
@@ -9,6 +9,10 @@ type MCPGroupSpec struct {
 	// Description provides human-readable context
 	// +optional
 	Description string `json:"description,omitempty"`
+
+	// RegisteredClients lists the client names that are registered with this group
+	// +optional
+	RegisteredClients []string `json:"registeredClients,omitempty"`
```

Comment:
why would we want to track clients on the k8s groups? this is not something we could do or enforce in k8s.